terraform/README.md

```diff
@@ -29,7 +29,7 @@ It is used to demonstrate migration path from TF 0.12 with Aiven provider 1.3.5
       │      │                           │     │     │                │         │
       │      │                         ┌─┘     │     │                │      ┌──┴──────┐
       │      │                         │       │     │                │      │         │
-      │      │                         │       │     └────Metrics─────┼──────┤  Redis  │
+      │      │                         │       │     └────Metrics─────┼──────┤  Redis® │
```

Comment:
If we're putting the ® after Redis, it also needs an asterisk. But we haven't put ® marks for the other products in this diagram, nor are there acknowledgements of their registration status.

Also, separate point, do we want Kibana at the top, or OpenSearch Dashboards? I think the terraform code actually refers to OpenSearch Dashboards now.

For the moment, I think it's best to just sort out Redis, as you've been doing, and I'll raise a new issue to look at checking that this page is up-to-date.